app/src/renderer/index.js

```diff
@@ -0,0 +1,2 @@
+// Initial welcome page. Delete the following line to remove it.
```

Comment:
Are u sure this file should be committed?

Reply:
It was removed